.github/workflows/pr-title-lint.yml

```
@@ -0,0 +1,15 @@
+name: PR Title Lint
+on:
+  pull_request:
+    # By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. We
+    # explicitly override here so that PR titles are re-linted when the PR text content is edited.
+    #
+    # Possible values: https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  pr-title-lint:
+    uses: linuxfoundation/lfx-ui/.github/workflows/_pr-title-lint.yml@main
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)